src/app/components/ConsentBanner/__snapshots__/index.test.jsx.snap

```diff
@@ -127,44 +211,103 @@ exports[`ConsentBannerButton should correctly render 1`] = `
   cursor: pointer;
 }
 
-.c0:focus,
-.c0:hover {
+.c4 button:focus,
+.c4 button:hover {
   color: #FFFFFF;
 }
 
-@media (min-width:37.5em) {
+.c4 a {
+  color: #F6A21D;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.c4 a:focus,
+.c4 a:hover {
+  color: #FFFFFF;
+}
+
+@media (min-width:25em) {
   .c0 {
+    padding: 1rem;
+  }
+}
+
+@media (min-width:37.5em) {
+  .c2 {
     font-size: 1.25rem;
     line-height: 1.5rem;
   }
 }
 
-<button
-  className="c0"
-  onClick={[Function]}
->
-  OK
-</button>
-`;
+@media (min-width:37.5em) {
+  .c2 {
+    width: 22%;
+    margin-right: 3.5%;
```

Comment:
WOW, if this wasn't copy and past it'd be a no

![](https://media1.giphy.com/media/fGv7XurtlWpVK/giphy.gif?cid=3640f6095c925fef49416d7936802ba3)